### ISSUE TITLE
Improve displaying of 'quoted' messages. 

### ISSUE THREAD

**Issue by sumanthvrao** [zulip/zulip-terminal#376]
The way ZT displays "quoted" text is being enclosing it as a single chunk of unindented text. It is difficult to distinguish from the quoted text and the body of the reply. There is further no distinction between different levels of quoting (Quoting a quoted message vs quoting a normal message) . 

We may want to look at the way web-app does this here, by using different levels of indentation to distinguish quoted messages from the replied body.

**Comment by sumanthvrao** [comment]
@zulipbot add "next major release"

**Comment by neiljp** [comment]
@sumanthvrao You're working on this now, right?

**Comment by sumanthvrao** [comment]
@neiljp Yes, I am working on this now. Could you please assign it to me?

**Comment by sumanthvrao** [comment]
The first of these - 'Indenting' quoted text based on level of quoting is now merged as e7421af798ada9a6729d5956d0cdeec9865ff9ae and e422fb89eb22f28b7dd08194f24795e3894da671 to reduce spacing at the start of the block-quote messages.
